src/deluge/gui/views/view.cpp

```diff
@@ -1952,7 +1952,7 @@ void View::drawOutputNameFromDetails(OutputType outputType, int32_t channel, int
 			break;
 		case OutputType::MIDI_OUT:
 			if (channel < 16) {
-				outputTypeText = "MIDI channel";
+				outputTypeText = "MIDI";
 			}
 			else if (channel == MIDI_CHANNEL_MPE_LOWER_ZONE || channel == MIDI_CHANNEL_MPE_UPPER_ZONE) {
 				outputTypeText = "MPE zone";
```

Comment:
should just be MPE - "zone" is analogous to "channel"

Reply:
Done good catch!